v2.0/delete.md

```diff
@@ -31,10 +31,12 @@ table td:first-child {
 | Parameter | Description |
 |-----------|-------------|
 |`relation_expr` | The name of the table you want to delete rows from.<br><br>Deleting from multiple tables in a single statement is not supported.|
-|`AS name` | Create an alias for the table name, completely hiding its original name. All subsequent references to the table must use its alias.<br><br>Aliases are primarily used with `JOIN`, which is not yet supported but is coming in a [future release](https://github.com/cockroachdb/cockroach/issues/2970).|
+|`AS table_alias_name` | Create an alias for the table name, completely hiding its original name. All subsequent references to the table must use its alias.<br><br>Aliases are primarily used with `JOIN`, which is not yet supported but is coming in a [future release](https://github.com/cockroachdb/cockroach/issues/2970).|
 |`WHERE a_expr`| `a_expr` must be an expression that returns Boolean values using columns (e.g. `<column> = <value>`). Delete rows that return `TRUE`.<br><br/>__Without a `WHERE` clause in your statement, `DELETE` removes all rows from the table.__|
+| `ORDER BY sortby_list` | Sort retrieved rows in the order of comma-separated column names you include in `sortby_list`. You can optionally specify `ASC` or `DESC` order for each column.<br><br>When <code>ORDER BY</code> is not included in a query, rows are not sorted by any consistent criteria. Instead, CockroachDB returns them as the coordinating node receives them.|
+| `LIMIT count` | Only retrieve `count` number of rows. <br><br>CockroachDB also supports `FETCH FIRST count ROWS ONLY` and `FETCH NEXT count ROWS ONLY` as aliases for `LIMIT`. If `count` is omitted, then one row is fetched.|
```

Comment:
Let's say `For compatibility with PostgreSQL, CockroachDB also supports...`

Reply:
Done.